### DISCUSSION TITLE
Is it possible to get a script for NetAlertX? The replacement for Pi.Alert project

### DISCUSSION THREAD

**Discussion by Luda83** [tteck/Proxmox#3946] [General] [answered]
Is it possible to get a script for NetAlert X? it is the successor to Pi.Alert. 

The active project has been renamed to NetAlertX and moved: jokob.sk/NetAlertX 

https://github.com/jokob-sk/Pi.Alert

[github.com/jokob-sk/NetAlertX](https://github.com/jokob-sk/NetAlertX)

**Comment by leiweibau** [comment, marked as answer]
Even if I am not the addressee, I would like to take this opportunity to say that NetAlertX is not necessarily the sole successor to "pucherot/Pi.Alert", which has been inactive for over 3 years. For more than 2 years I have been maintaining a fork that has also evolved and I am very flattered that @tteck has included my fork in his script collection. It is not the inactive project of pucherot but mine, which is offered here and NetAlertX is NOT the successor. I just wanted to put this on record, because I have invested a lot of time and effort in the last few years, which is always “wiped away” with the reference to the old project.

**Reply by Luda83** [reply]
You might want to take that up with google then so you get the credit you deserve. I never heard of Pi.Alert until 2 weeks ago on reddit. When i googled Pi.Alert the "pucherot/Pi.Alert" project comes up first showing it has not been updated since 2021. The third result was for the old page of "jokob-sk/Pi.Alert" project which says "The active project has been renamed to NetAlertX and moved: jokob.sk/NetAlertX" So as someone who never knew the past of the project i had no idea you had a version. Thank you for letting me know. I will definitely try your version then. And thank you @tteck for your amazing scripts!

**Reply by leiweibau** [reply]
All good. I am aware of the circumstances with the search engines, so I also understand why you have come to this conclusion, especially as a newcomer to this topic. The strong presence of the Docker version in the search engines also favors the perception that you have also come to. Unless you've been involved in the topic for a while, you wouldn't know that the statement about the successor refers to its own version, which was also previously called Pi.Alert. There is also no ill will between jokob-sk and myself, we give each other tips from time to time on problems we come across.

**Reply by Luda83** [reply]
Awesome. I am going to close this discussion. I am currently resetting up two of my proxmox servers from scratch and will definitely use @tteck script with your version of Pi.Alert!

**Reply by tteck** [reply]
Currently, there are no plans to add NetAlertX.

**Reply by Luda83** [reply]
That is fine, i am glad i found out their is a active Pi.Alert project. Again @tteck , Love the scripts!! Keep up the good work!

**Reply by daubman** [reply]
@tteck Is that due to its tight docker integration or some other reason(s)? I am curious because NetAlertX seems to have support for [Omada integration](https://github.com/jokob-sk/NetAlertX/tree/main/front/plugins/omada_sdn_imp) (and a lot of other things via some great plugins) missing from Pi.Alert that I would love to use - but I'd much rather do so via one of your awesome LXC scripted ports :-)
